src/content/profile-data.js

```diff
@@ -821,9 +821,11 @@ export function getTracingMarkers(thread: Thread): TracingMarker[] {
   return tracingMarkers;
 }
 
-export function filterTracingMarkersToRange(tracingMarkers: TracingMarker[],
-                                            rangeStart: number,
-                                            rangeEnd: number): TracingMarker[] {
+export function filterTracingMarkersToRange(
+  tracingMarkers: TracingMarker[],
+  rangeStart: number,
+  rangeEnd: number
+): TracingMarker[] {
```

Comment:
This change comes from the eslint upgrade I started, but I didn't revert it when I reverted the eslint upgrade. Tell me if you prefer that I revert this.